dumps/obj_fetch_non-key.c

```diff
@@ -0,0 +1,218 @@
+                        dcs_csum_info_list_init
+                        for (i = 0; i < iod_nr; i++) {
+                                bsgl = bio_iod_sgl(ioc->ic_biod, i);
+                                bio_sgl_init(bsgl, iov_nr);
+                vos_dth_set -> tls->vtl_dth = NULL;
+                vos_ts_set_add(ioc->ic_ts_set, ioc->ic_cont->vc_ts_idx, NULL, 0); (if (!vos_ts_in_tx(ts_set)) return 0;)
+                occ = vos_obj_cache_current()
+                        vos_obj_cache_get (return vos_tls_get(standalone)->vtl_ocache;)
+                vos_obj_hold(occ, …)
+                        daos_lru_ref_hold(lcache = occ, …)
+                                link = d_hash_rec_find(&lcache->dlc_htable, key, key_size);
+                                        idx = ch_key_hash(htable, key, ksize);
+                                        bucket = &htable->ht_buckets[idx];
+                                        ch_bucket_lock(htable, idx, !is_lru);
+                                        link = ch_rec_find(...)
+                                        /* !!! */
+                                        if (link != NULL)
+                                                ch_rec_addref(htable, link);
+                                        ch_bucket_unlock(htable, idx, !is_lru);
+                                if (link != NULL) {
+                                        llink = link2llink(link);
+                                         if (!d_list_empty(&llink->ll_qlink))
+                                                d_list_del_init(&llink->ll_qlink);
+                        /* rc == 0 */
+                        /** Object is in cache */
+                        obj = container_of(lret, struct vos_object, obj_llink);
+                        if (obj->obj_df) { /* Persistent memory address of the object */
+                                tmprc = vos_ilog_ts_add(ts_set, &obj->obj_df->vo_ilog, &oid, sizeof(oid)); /* if (!vos_ts_in_tx(ts_set)) return 0;*/
+                                goto check_object
+                        if (!create) {
+                                vos_ilog_fetch
+                                        vos_ilog_fetch_internal(umm, coh, intent, ilog, &epr, bound, has_cond, punched, parent, info);
+                                                vos_ilog_desc_cbs_init(&cbs, coh);
+                                                ilog_fetch(umm, ilog, &cbs, intent, has_cond, &info->ii_entries);
+                                                        if(ilog_fetch_cached(umm, root, cbs, intent, has_cond, entries)) true
+                                                                ilog_status_refresh()
+                                                                return true /* DIFFERENT */
+                                                        return 0 (rc);
+
+                                                /* !ilog_empty(root)) */
+#if 0 // It is skipped
```

Comment:
It seems you are comparing here two sequences but the first one is not disclosed. I understand this part is skipped compared to the first one, is it correct?